### DISCUSSION TITLE
I want to change the Backspace that triggers the onNodesDelete event to Delete.

### DISCUSSION THREAD

**Discussion by xblackh** [xyflow/xyflow#3635] [Q&A] [answered]
I want to change the Backspace that triggers the onNodesDelete event to Delete.

**Comment by bcakmakoglu** [comment, marked as answer]
You can set the key code by using the [`deleteKeyCode`](https://reactflow.dev/api-reference/react-flow#delete-key-code) prop.

```tsx
<ReactFlow nodes={nodes} edges={edges} deleteKeyCode={'Delete'}>...</ReactFlow>
```

**Reply by xblackh** [reply]
thank you👍
